### PR TITLE
[Improvement](doc) add table-buckets doc for flink connector

### PR DESCRIPTION
**Pull request by vinlee19**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -512,7 +512,7 @@ insert into doris_sink select id,name,bank,age from cdc_mysql_source;
 | --postgres-conf         | Postgres CDCSource 配置，例如--postgres-conf hostname=127.0.0.1 ，您可以在[这里](https://ververica.github.io/flink-cdc-connectors/master/content/connectors/postgres-cdc.html)查看所有配置Postgres-CDC，其中hostname/username/password/database-name/schema-name/slot.name 是必需的。 |
 | --sqlserver-conf        | SQLServer CDCSource 配置，例如--sqlserver-conf hostname=127.0.0.1 ，您可以在[这里](https://ververica.github.io/flink-cdc-connectors/master/content/connectors/sqlserver-cdc.html)查看所有配置SQLServer-CDC，其中hostname/username/password/database-name/schema-name 是必需的。 |
 | --sink-conf             | Doris Sink 的所有配置，可以在[这里](https://doris.apache.org/zh-CN/docs/dev/ecosystem/flink-doris-connector/#%E9%80%9A%E7%94%A8%E9%85%8D%E7%BD%AE%E9%A1%B9)查看完整的配置项。 |
-| --table-conf            | Doris表的配置项，即properties中包含的内容。 例如 --table-conf replication_num=1 |
+| --table-conf            | Doris表的配置项，即properties中包含的内容（其中table-buckets例外，非properties属性）。 例如 --table-conf replication_num=1， 而 --table-conf table-buckets=10用于指定建表时buckets数量。 |
```
Comment (JNSimba) [inline docs/zh-CN/docs/ecosystem/flink-doris-connector.md:515]: Can we give an example to illustrate the usage of table-buckets?

Comment (vinlee19) [reply docs/zh-CN/docs/ecosystem/flink-doris-connector.md:515]: I will add relevant usage examples later

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]

**Comment by vinlee19** [comment]
run buildall

**Pull request merged**